[2025-02-13T00:04:01.018+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-13T00:04:01.029+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_omdb_api.teste scheduled__2025-02-13T00:03:00+00:00 [queued]>
[2025-02-13T00:04:01.034+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_omdb_api.teste scheduled__2025-02-13T00:03:00+00:00 [queued]>
[2025-02-13T00:04:01.035+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-13T00:04:01.044+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): teste> on 2025-02-13 00:03:00+00:00
[2025-02-13T00:04:01.049+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=955) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-13T00:04:01.050+0000] {standard_task_runner.py:72} INFO - Started process 956 to run task
[2025-02-13T00:04:01.050+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_omdb_api', 'teste', 'scheduled__2025-02-13T00:03:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/call_api.py', '--cfg-path', '/tmp/tmp3pmgq8cl']
[2025-02-13T00:04:01.051+0000] {standard_task_runner.py:105} INFO - Job 42: Subtask teste
[2025-02-13T00:04:01.083+0000] {task_command.py:467} INFO - Running <TaskInstance: call_omdb_api.teste scheduled__2025-02-13T00:03:00+00:00 [running]> on host 76fc25d1410f
[2025-02-13T00:04:01.138+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Celso Cazetto' AIRFLOW_CTX_DAG_ID='call_omdb_api' AIRFLOW_CTX_TASK_ID='teste' AIRFLOW_CTX_EXECUTION_DATE='2025-02-13T00:03:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-13T00:03:00+00:00'
[2025-02-13T00:04:01.139+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-13T00:04:01.149+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-13T00:04:01.150+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'source /home/***/venv/omdb_env/bin/activate && python /opt/***/pipeline/main.py']
[2025-02-13T00:04:01.157+0000] {subprocess.py:99} INFO - Output:
[2025-02-13T00:04:01.986+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:01,986 - INFO - ESTOU RODANDO ESSA PORRA
[2025-02-13T00:04:01.987+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:01,987 - INFO - Carregando o arquivo de configuração
[2025-02-13T00:04:01.988+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:01,987 - INFO - Buscando os dados de filme do ID:tt0371746
[2025-02-13T00:04:02.100+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,100 - INFO - Dados da API carregados com sucesso
[2025-02-13T00:04:02.101+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,100 - INFO - Iniciando transformação dos dados.
[2025-02-13T00:04:02.108+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,108 - INFO - Estabelecendo conexão com o banco de dados:
[2025-02-13T00:04:02.137+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,136 - INFO - Conexão estabelecida com sucesso.
[2025-02-13T00:04:02.154+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,154 - INFO - Iniciando o inserção na tabela: omdb.filmes
[2025-02-13T00:04:02.157+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,157 - INFO - 0 Inseridos na tabela omdb.filmes
[2025-02-13T00:04:02.160+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,160 - INFO - Iniciando o inserção na tabela: omdb.genero
[2025-02-13T00:04:02.162+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,162 - INFO - 0 Inseridos na tabela omdb.genero
[2025-02-13T00:04:02.169+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,169 - INFO - Iniciando o inserção na tabela: omdb.filme_genero
[2025-02-13T00:04:02.171+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,171 - INFO - 0 Inseridos na tabela omdb.filme_genero
[2025-02-13T00:04:02.172+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,172 - INFO - Conexão finalizada com o banco
[2025-02-13T00:04:02.173+0000] {subprocess.py:106} INFO - 2025-02-13 00:04:02,172 - INFO - Fim do processo de ingestão.
[2025-02-13T00:04:02.173+0000] {subprocess.py:106} INFO - 2.0.38
[2025-02-13T00:04:02.255+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-13T00:04:02.275+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-13T00:04:02.276+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=call_omdb_api, task_id=teste, run_id=scheduled__2025-02-13T00:03:00+00:00, execution_date=20250213T000300, start_date=20250213T000401, end_date=20250213T000402
[2025-02-13T00:04:02.306+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-13T00:04:02.320+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-13T00:04:02.322+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
